Makefile

```diff
@@ -42,7 +42,9 @@ patch-nvml-h: $(PKG_BINDINGS_DIR)/nvml.h
 $(PKG_BINDINGS_DIR)/nvml.h: $(GEN_BINDINGS_DIR)/nvml.h | $(PKG_BINDINGS_DIR)
 	sed -E 's#(typedef\s+struct)\s+(nvml.*_st\*)\s+(nvml.*_t);#\1\n{\n    struct \2 handle;\n} \3;#g' $(<) > $(@)
 
-bindings: $(PKG_BINDINGS_DIR)/nvml.h $(SOURCES) | $(PKG_BINDINGS_DIR)
+bindings: .create-bindings .strip-autogen-comment
+
+.create-bindings: $(PKG_BINDINGS_DIR)/nvml.h $(SOURCES) | $(PKG_BINDINGS_DIR)
```

Comment:
What does the `|` do exactly? I've never used that before myself.

Reply:
It's an ordering dependency. I suppose in this case it's not strictly needed and a normal dependency would work.

https://www.gnu.org/software/make/manual/html_node/Prerequisite-Types.html